spec/features/ckey2bibframe_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+describe 'Do a conversion' do
+  before do
+    marc = '00924ccm a2200265   450000100040000000300060000400500170001000800410'\
+           '00270350023000680350023000910400010001141000043001242450053001672600'\
+           '05600220300001800276500001900294500005000313500003700363505004500400'\
+           '596000700445650006800452998001000520918000800530999012000538123SIRSI'\
+           '19900820141050.0731212|||||||||fr            ||| | fre    a(OCoLC-M)'\
+           '123010268  a(OCoLC-I)268806670  dOrLoB1 aAmellér, André,d1912-1990=^'\
+           'A36038610aA la francaise;c3 pieces pour quintette a vent.  aParis, E'\
+           'ditions Musicales Transatlantiques,cc1973.  a5 pts.c32cm.  aCaption '\
+           'title.  aFor flute, oboe, clarinet, horn, and bassoon.  aDuration: a'\
+           'bout 13 min., 30 sec.0 a1.Prelude, Fughetto.- 2.Grave.- 3.Rondo.  a1'\
+           '8 0aWind quintets (Bassoon, clarinet, flute, horn, oboe)=^A1075272  '\
+           'aSCORE  a123  aM557 .A498 A1 PTSwLCc1i36105041841193d7/13/2012e6/6/2'\
+           '012j5lSCORESmMUSICn4p$40.00rYsYtSCOREu12/12/1973'
+
+    allow_any_instance_of(Ckey2bibframe).to receive(:catalogdump).and_return(marc)
+  end
+
+  # Skip running this test until the xform-marc21-to-xml jar artifact can be put on github
+  xit 'Does a conversion and displays a page with a conversion result' do
+    visit new_ckey2bibframe_path
+
+    fill_in 'ckey2bibframe_ckey', with: '123'
+    fill_in 'ckey2bibframe_baseuri', with: 'http://ld4p.stanford.edu'
+
+    click_button 'Do conversion'
+
+    expect(page).to have_css('h1', text: 'Ckey to Bibframe2 Conversion')
+    expect(page).to have_css('h3', text: 'MarcXML')
+    expect(page).to have_css('h3', text: 'Bibframe2')
+    expect(page).to have_css('pre', text: '<?xml version="1.0" encoding="UTF-8"?>')
+    expect(page).to have_css('pre', text: '<marcxml:controlfield tag="001">123</marcxml:controlfield>')
+    expect(page).to have_css('pre', text: '<bf:Work rdf:about="http://ld4p.stanford.edu123#Work">')
+  end
```

Comment:
This spec is much better :+1:   Only one suggestion - if the `ld4p.stanford.edu` prefix is moved to a settings constant, it could be used here in the spec to identify the bf:Work.